src/logger.ts

```diff
@@ -0,0 +1,22 @@
+import bunyan from 'bunyan';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const log = bunyan.createLogger({
+  name: 'fluxity-app',
+  level: bunyan.TRACE,
```

Comment:
change level based on NODE_ENV. if it's on `production`, then put this to bunyan.WARN, if not, then bunyan.TRACE